grpc-ingest/src/prom.rs

```diff
@@ -63,6 +63,11 @@ lazy_static::lazy_static! {
         "download_metadata_inserted_count", "Total number of inserted tasks for download metadata"
     ).unwrap();
 
+    static ref DOWNLOAD_METADATA_FAILURE_COUNT: IntGaugeVec = IntGaugeVec::new(
+        Opts::new("download_metadata_failure_count", "Status of failed tasks for download metadata"),
```

Comment:
This metric name needs to be changed. Generally us past tense for the metrics
`download_metadata_failed_count`